docs/q-and-a.md

```diff
@@ -6,8 +6,12 @@ The Q&A frontend is a finder-frontend feature that walks users through the facet
 
 Any, all or no filters can be selected for each facet, and any question may be skipped. At the end of the process, the user is directed to the given finder with their selections intact.
 
-At the time of writing, this feature has been defined for one finder and relies on a [YAML file](../lib/prepare_business_uk_leaving_eu.yaml). This file defines things like the base path of the Q&A and the underlying finder, as well as questions, question types and descriptions for each question page.
+It emulates the look and feel of a [Smart Answer](https://github.com/alphagov/smart-answers), but the code lives here so that we can use the user's answers to pre-populate the finder.
```

Comment:
I'm not sure why the code was added here. I think it's because the Q&A uses features that smart-answers don't have, like nested checkboxes, and not to pre-populate the finder as you don't need to be in the same app to do that.

Reply:
Thanks - I did wonder! But a better solution would probably have been to extend smart-answer to be able to use things like nested checkboxes. 🤔 

I wonder if the reason it's here is actually because it means all the config for the business finder is in one repo? (The finder and the Q&A in one place). Probably was the case before the config 'spread' into search-api too.

Reply:
It was probably added here for speed and ease. Smart-answers is not any easy app to add features to.

Reply:
Smart-answers are out of date style-wise, so I'm hoping that the it was the question page pattern from the design pattern that was followed: https://design-system.service.gov.uk/patterns/question-pages/

@DilwoarH was on the team and might know more.

However I don't know, so rather than guessing and being wrong, I think we should remove the paragraph.

Reply:
Smart answers was an option but it did not provide the specific functionality that we needed. 

We needed to be able to take the answers from a set of questions then create a URL with query strings. Smart answers only pointed users to a specific link or gave them guidance based on a specific set of answers - therefore smart answers wasn't really useful.

There is also other logic such as nested checkboxes as mentioned by @leenagupte which did not exist in smart answers.

Hope this helps.

Reply:
Have tweaked the wording to make that clearer. Thanks 👍